package.json

```diff
@@ -129,6 +129,7 @@
     "console-taskgraph": "^1.6.0",
     "cronstrue": "^1.82.0",
     "cross-env": "^6.0.3",
+    "deep-sort-object": "^1.0.2",
```

Comment:
Is this to avoid spurious changes in the generated JSON?  We have `json-stable-stringify` for that purpose already, so perhaps we can just reuse that..